src/challenges/challengeTwo/IntegrationComponent/IntegrationComponent.test.tsx

```diff
@@ -1,8 +1,56 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { IntegrationComponent } from "./IntegrationComponent";
-
-// describe("IntegrationComponent", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import axios from "axios";
+import { IntegrationComponent } from "./IntegrationComponent";
+
+jest.mock("axios");
+
+describe("IntegrationComponent", () => {
+  it("performs user journey successfully", async () => {
```

Comment:
Firstly I would change the test name, what user journey? What is happening? Your tests are a documentation of the features of the component. It's like writing a test to say "it works successfully". Not very helpful. 

I would move the mocks and setup out the test, as I showed above in a previous file. 

Maybe also separate out the tests into ones that build on each other.